### DISCUSSION TITLE
[Porousflow][mining] modelling water seepage and filling the pit

### DISCUSSION THREAD

**Discussion by Traiwit** [idaholab/moose#20768] [Q&A Modules: General] [answered]
Hi MOOSE expert,

I am trying to model modelling water seepage into a pit (empty space) and then slowly filling the pit.
![image](https://user-images.githubusercontent.com/79562439/162872626-bae4c007-c885-4cce-a033-940838f388a4.png)
just wondering if anyone has done this before?

so the initial condition would be right at the end of pit excavation:
![image](https://user-images.githubusercontent.com/79562439/162872857-2774ca48-dc34-45fc-b78b-6b4404850a62.png)

then I remove the drainage surface bc, and just run an equilibrium simulation where BCs are applied to the 4 sides of the domain (right left front back)

I tried what I mentioned above with very high perm, the pit is filled very quickly in a matter of days, which isn't realistic:
![image](https://user-images.githubusercontent.com/79562439/162873919-d116c590-fd11-4893-9974-0c57cb85dffa.png)
(after 5 days)

The question would be what should be the value of the empty pit material properties (perm and porosity)? porosity I guess it should be 1, but what would be the appropriate perm? as this will affect the rate of pit filling
and just wondering if this is the right approach to modelling this kinda seepage? 

Kind regards,
Traiwit

I will tag the hydro expert here: @WilkAndy

**Comment by WilkAndy** [comment, marked as answer]
I agree that the porosity should be ~1 and the perm should be "large": probably 1000 times larger than your largest perm is a good choice.

The pit shouldn't fill that fast.  Are you using unsaturated physics, or fully-saturated physics?

**Reply by Traiwit** [reply]
Hi @WilkAndy,

I'm using fully-saturated physics for drainage (due to convergence, and Abaqus comparison), hence I used it for seepage too. Should I switch to unsaturated in this case?
```
[Modules]
  [./FluidProperties]
    [./the_simple_fluid]
      type = SimpleFluidProperties
    [../]
  [../]
[]
[PorousFlowFullySaturated]
  porepressure = porepressure
  coupling_type = Hydro
  gravity = '0 0 -9.81'
  fp = the_simple_fluid
  time_unit = days
  add_darcy_aux = false
[]
```

**Reply by WilkAndy** [reply]
Definitely you need `PorousFlowUnsaturated`.  At the moment, your pit initially contains water at zero porepressure.  That's why it doesn't take long to "fill": your definition of "fill" is just moving a bit of water into the pit to increase porepressure to hydrostatic.

**Reply by Traiwit** [reply]
Hi @WilkAndy, this actually solves the problem. Thank you!

```
[PorousFlowUnsaturated]
  porepressure = porepressure
  coupling_type = Hydro
  gravity = '0 0 -9.81'
  fp = the_simple_fluid
  relative_permeability_exponent = 2
  relative_permeability_type = Corey
  residual_saturation = 0.01
  van_genuchten_alpha = 1E-6
  van_genuchten_m = 0.6
  time_unit = years
[]
```

this is the pwp after 2 years
![image](https://user-images.githubusercontent.com/79562439/162885192-88f18f0e-73c7-43fa-a68b-5435d2f5ff99.png)
